config/config.go

```diff
@@ -664,6 +671,33 @@ func (c *KubernetesSDConfig) UnmarshalYAML(unmarshal func(interface{}) error) er
 	return checkOverflow(c.XXX, "kubernetes_sd_config")
 }
 
+// EC2SDConfig is the configuration for EC2 based service discovery.
+type EC2SDConfig struct {
+	Region          string             `yaml:"region"`
+	AccessKey       string             `yaml:"access_key,omitempty"`
+	SecretKey       string             `yaml:"secret_key,omitempty"`
```

Comment:
You requested making sure this was in the regex to hide stuff. I was not able to find that regex or any reference to hiding. Could you point me to what you're talking about so I can add it?